docs/examples/revocation/2. Issue Credential.md

```diff
@@ -0,0 +1,96 @@
+# When issuing revocable credentials
```

Comment:
It's effectively identical to standard credential issuance process -- should be merged with Issuing Credentials sections